src/tools/compiletest/src/header.rs

```diff
@@ -935,16 +935,26 @@ struct HeaderLine<'ln> {
 pub(crate) struct CheckDirectiveResult<'ln> {
     is_known_directive: bool,
     directive_name: &'ln str,
+    trailing_directive: Option<&'ln str>,
 }
 
 // Returns `(is_known_directive, directive_name)`.
```

Comment:
This comment is outdated, probably just delete it.

Reply:
Sure, removed.